### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by cbalster**
I'm aware we don't use proguard right now, but we probably should at some point

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -39,24 +38,24 @@ android {
...
-    apt "com.jakewharton:butterknife-compiler:$butterknife_version"
+    annotationProcessor "com.jakewharton:butterknife-compiler:$butterknife_version"
 
     compile 'org.ligi:AXT:0.35'
     compile 'info.metadude.android:typed-preferences:2.1.0'
     compile "com.google.dagger:dagger:$dagger_version"
 
     provided 'javax.annotation:jsr250-api:1.0'
-    apt "com.google.dagger:dagger-compiler:$dagger_version"
+    annotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
```
Comment (ligi) [inline app/build.gradle]: interesting - is this now coming from the android-build-plugin?

File: build.gradle [hunk 2]
```diff
@@ -1,21 +1,22 @@
 buildscript {
     ext {
-        support_version = "24.2.1"
-        dagger_version = "2.6.1"
+        support_version = "25.1.0"
+        dagger_version = "2.8"
```
Comment (ligi) [inline build.gradle]: would be OK with this - just as a note - currently Kotlin and dagger 2.8 are not playing well together - so I usually stick to 2.7 currently

File: app/build.gradle (hunk 1 above)
Comment (johnjohndoe) [inline app/build.gradle]: Yes. @hvisser announced it [here](http://www.littlerobots.nl/blog/Whats-next-for-android-apt/).

File: build.gradle (hunk 2 above)
Comment (cbalster) [inline build.gradle]: thanks, good to know :)

File: app/build.gradle (hunk 1 above)
Comment (ligi) [inline app/build.gradle]: nice - thanks

**Comment by ligi** [comment]
:+1:

**Pull request merged**